iosApp/iosApp/Pages/NearbyTransit/NearbyTransitView.swift

```diff
@@ -51,10 +48,7 @@ struct NearbyTransitView: View {
         .onAppear {
             getGlobal()
             getNearby(location: location, globalData: globalData)
-            Task {
-                await getPredictionsFeatureFlag()
-                joinPredictions(state.nearbyByRouteAndStop?.stopIds())
-            }
+            joinPredictions(state.nearbyByRouteAndStop?.stopIds())
```

Comment:
Should this still be performed in a `Task`?

Reply:
Oh good Q, it [originally wasn't](https://github.com/mbta/mobile_app/pull/423/files#diff-a02e18ebdb7fec9ec2891eb904127b4db46b407ddc37d170333bd776f1448113L51), but I moved it into a task so that it would be run after the feature flag status was known. I was assuming that waiting for a success / error on join was already done asynchronously, but will verify whether that is the case.

Reply:
I updated the `connect` function to include the following 
```
  ?.receive(PhoenixPushStatus.Ok) { message ->
                runBlocking {
                    print("Wait for 5sec")
                    delay(5000)
                    print("Done waiting for 5sec")
                }
                println("joined channel ${message.subject}")
                handleV2JoinMessage(message, onJoin)
            }
```

And confirmed that does not block the call to update nearby transit. Logs:
```
STARTING JOIN 2024-10-07 21:02:05 +0000
ENDING JOIN 2024-10-07 21:02:05 +0000
Wait for 5sec
UPDATE NEARBY ROUTES 2024-10-07 21:02:05 +0000
UPDATE NEARBY ROUTES 2024-10-07 21:02:07 +0000
Done waiting for 5sec
joined channel predictions:stops:v2:9206,9223,place-WR-0062,19573,5985
Received 13 predictions on join
```

Reply:
Perfect, thanks for checking!